Example/README.md

```diff
@@ -0,0 +1,37 @@
+# FaunaDB Driver Example Project
+
+This is a simple CRUD project we made in order to show how to use the Swift
+driver along with a iOS project.
+
+## Requirements
+
+- [CocoaPods](https://cocoapods.org/)
+
+## Running the example
+
+- Run `pod install` to setup the project
+- Using XCode 8+, open the workspace `Example.xcworkspace`
+- Edit (or duplicate) the scheme "Example" setting your key's secret at the
+  environment variable "FAUNA_ROOT_KEY"
+
+## Implementation
+
+This demo is a simple blog post manager. It has a basic model called `Post` that
+must have a title and optionally have a body.
+
+Most of Fauna's related code is located at [Posts.swift](https://github.com/faunadb/faunadb-swift/blob/master/Example/Example/Post.swift)
```

Comment:
I'd say "most of the interaction with FaunaDB" instead, it calls out specifically what most people will be looking for.